data/cmake/FindOdinData.cmake

```diff
@@ -46,9 +46,11 @@ ENDIF(PkgConfig_FOUND)
 if (NOT APPLE)
     set(LIB_RECEIVER FrameReceiver)
     set(LIB_PROCESSOR FrameProcessor)
+	set(LIB_SIMULATOR FrameSimulator)
 else(NOT APPLE)
```

Comment:
Are lines 49 and 53 indented to the same level as the rest of their respective paragraphs? In the browser it doesn't appear to be that way. Nitpicking: Line 53 "SET" in capitals whereas the above two lines the same keyword is in lower case.

Reply:
For some reason, VS Code doesn't do the normal "put 4 spaces in when you hit the tab key" thing when editing cmake files, hence the odd looking indentation. Fixed it